.travis.yml

```diff
@@ -32,8 +31,18 @@ before_deploy:
 - chmod 600 .utility/loklakserver.1
 - ssh-add .utility/loklakserver.1
 deploy:
-  provider: script
-  skip_cleanup: true
-  script: ssh loklak@104.155.231.103 /opt/loklak/automatic-deploy.sh
-  on:
-    branch: master
+  - provider: script
+    skip_cleanup: true
+    script: ssh loklak@104.155.231.103 /opt/loklak/automatic-deploy.sh
```

Comment:
@yukiisbored Please help to avoid dependencies on outside services and keep deploy scripts in the  repo itself. Thank you.